custom_components/spotcast/manifest.json

```diff
@@ -5,8 +5,8 @@
   "requirements": [
     "spotify_token==1.0.0"
   ],
-  "homeassistant": "0.113.0",
-  "version": "3.4.7",
+  "homeassistant": "2021.4.0",
```

Comment:
In says nothing about "homeassistant" field:
https://developers.home-assistant.io/docs/creating_integration_manifest/

I think it should go to `hacs.json`.